ddpui/celeryworkers/tasks.py

```
@@ -1123,3 +1178,10 @@ def setup_periodic_tasks(sender: Celery, **kwargs):
             compute_dataflow_run_times.s(),
             name="compute run times of each deployment based on its past flow runs",
         )
+
+    # sync airbyte job stats for connections; every 24 hours
+    sender.add_periodic_task(
+        crontab(minute=0, hour=0),
+        sync_airbyte_job_stats_for_connections.s(),
```

Ruff (0.11.9)

Undefined name `sync_airbyte_job_stats_for_connections`

(F821)